Documentation/VSCodeConfiguration.md

```diff
@@ -168,121 +178,123 @@ Note: The Assertion und KUBSan Problem matchers will only run after you have clo
             "label": "build",
             "type": "shell",
             "command": "bash",
-                "args": [
-                    "-c",
-                    "Meta/serenity.sh build ${input:arch} ${input:compiler}"
-                ],
-                "options": {
-                    "env": {
-                        "SERENITY_RAM_SIZE": "4G",
-                    }
+            "args": [
+                "-c",
+                "Meta/serenity.sh build ${input:arch} ${input:compiler}"
+            ],
+            "options": {
+                "env": {
+                    // Put your custom run configuration here, e.g. SERENITY_RAM_SIZE
```

Comment:
What I meant, is that the build task does not need an env set at all

Reply:
Removed, understandable.